src/style.css

```diff
@@ -1,3 +1,82 @@
+:root {
+  --light-blue: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.header {
+  box-sizing: border-box;
+  padding-left: 50px;
+  padding-right: 50px;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+}
+
+.header-logo {
+  display: block;
+}
+
+.nav__list {
+  padding-inline-start: 0;
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). Style all elements using '.class' and if needed with ':pseudo-class', 'pseudo-element' and '[attribute]'.